npm/src/tsconfig.json

```diff
@@ -2,11 +2,11 @@
   "compilerOptions": {
     "module": "node16",
     "target": "ES2020",
-    "lib": ["DOM", "WebWorker", "ES2020"],
```

Comment:
Was this breaking anything, or just clean up? I think it was added as the npm package also targets browser, and we may at some point use a WebView in VS Code for visualizations. Harmless to remove it if not needed now I guess, as long as anybody who needs it later knows where to go put it back :-)

Reply:
Yes, as soon as I turned off `skipLibCheck` I started getting duplicate definition errors across `DOM` and `WebWorker`. 

I decided to turn off `skipLibCheck` because I wanted to get errors from `qsc_wasm.d.ts` to catch some obvious mistakes I was making in the wasm TypeScript definitions.

I think this is "technically correct" today, since we run in the extension host worker, we don't have access to the DOM? If we added DOM manipulation would we add it in this package, or a different one? I'm content to leave that unanswered for now...